tests/tests_fabric/stress_test.sh

```diff
@@ -0,0 +1,40 @@
+test_batch_size="${PL_STANDALONE_TESTS_BATCH_SIZE:-6}"
```

Comment:
@robieta It is ok to run certain tests without batching. I've done the same for the parity tests and had to implement a retry mechansim (similar to what you have here): https://github.com/Lightning-AI/lightning/blob/master/tests/parity_fabric/run_standalone_tasks.sh

As you can see there, I'm running these ddp tests sequentially to avoid the batching (which is otherwise useful for other tests to speed up CI). 

Reducing the batch size here from 6 to 4 might be a temporary solution but I would suggest to disentangle your test case completely from how the others are run.